magento/inventory
# Add default Source in Data Install.

### Description


### Fixed Issues (if relevant)

1. magento-engcom/msi#41: Add default Source in Data Install

### Manual testing scenarios

1. Remove data version from table "setup_module", Magento_InventoryCatalog module
2. Run bin/magento setup:upgrade
3. Default source should be visible in table "inventory_source" with id: 1

### Contribution checklist
